index.php

```diff
@@ -0,0 +1,131 @@
+<?php
+require 'apicall.php';
+/**
+ * Class fetch to fetch the data from API.
+ * 
+ * Class for fetching title,service,image,alias and icon 
+ */
+class fetch
+{
+  public $title;
+  public $service;
+  public $image;
+  public $alias;
+  public $iconArr;
+  /**
+   * fetch constructor.
+   * 
+   * @param string $alias,$title,$service,$image,$iconArr
+   * 
+   * Fetched data will be passed as parameter.
+   */
+  function __construct($title, $service, $image, $alias, $iconArr)
+  {
+    $this->title = $title;
+    $this->service = $service;
+    $this->image = $image;
+    $this->alias = $alias;
+    $this->iconArr = $iconArr;
+  }
+  /**
+   * iconArrLen function to count the number of icons which will be stored in array format.
+   *
+   * @return integer
+   */
+  function iconArrLen()
+  {
+    return count($this->iconArr);
+  }
+}
+// Initializing an array to contain all retrieved data.
+$fullArr = [];
+// Calling apiCall function to call the api and retrieve data.
+$data = apiCall("https://www.innoraft.com/jsonapi/node/services");
+// Declaring domain name to concate it wherever needed.
+$domain = "https://www.innoraft.com";
+
+for ($i = 0; $i < count($data['data']); $i++) {
+  if ($data['data'][$i]['attributes']['field_services']['value'] != NULL && $i > 11) {
```

Comment:
Would it be possible to store common indexes used across the file in a variable to make the code more easily readable and easy to handle.

### One such example:
We can put this in a variable and use it wherever applicable.
`$data['data'][$i]['attributes']`